meta/client/client.go

```diff
@@ -16,7 +16,7 @@ type Client interface {
 	GetAllObject(bucketName, objectName, version string) (object []*Object, err error)
 	PutObject(object *Object, tx interface{}) error
 	UpdateAppendObject(object *Object, tx interface{}) error
-	UpdateObjectAttrs(object *Object) error
+	UpdateObjectName(object *Object, sourceObject string) error
```

Comment:
Why don't u define the function name as "RenameObject" to make it consistent with the func above.

Reply:
Have changed